app/services/foreman/renderer/scope/macros/helpers.rb

```diff
@@ -108,6 +108,26 @@ def shell_escape(string)
           def current_date(format: '%F')
             Time.zone.today.strftime(format)
           end
+
+          apipie :method, 'Checks whether a value is truthly or not' do
+            optional :value, Object, desc: 'Value to check'
+            returns one_of: [true, false], desc: 'Returns true if the value can be considered as truthly, false otherwise'
+            example "is_true?('1') #=> true"
+            example "is_true?('0') #=> false"
+          end
+          def is_true?(value = nil)
+            Foreman::Cast.to_bool(value) == true
```

Comment:
If it casts the object to a bool then there's no need to compare it

Reply:
I don't mind it being bit explicit :shrug: